src/modules/common/renderHtml.js

```diff
@@ -10,6 +10,11 @@ const renderJS = (files: PageResourceCacheStateType) => `
   var hrefs = document.querySelectorAll('[href]')
   var srcs = document.querySelectorAll('[src]')
   var files = ${JSON.stringify(files)}
+    
+    if (!files) {
+      console.warn('Files for WebView undefined! Not replacing urls...' )
```

Comment:
What would happen here if you use the js api: alert(...)? :thinking: 

It would be cool if a react-native or some other alert would pop up if errors occur. Maybe also surround everyting with a try catch to propagate errors to the UI.

Right now this warning is basically useless as no one (not even all developers) can see this message.

Reply:
There is this stone-age issue: https://issues.integreat-app.de/browse/NATIVE-9

If the alert would give feedback to the user then we can close this I think :)

Reply:
I'll try. If it's not working, I can also just as well forward all errors (and therefore complete NATIVE-9). Shouldn't be that much of a hassle.

Reply:
If it does not work right away then skip this.

We'll prioritise NATIVE-9 if we think we need it.

Reply:
Ok. `alert()` works. Do we want to display an error message that is useful for the end user? Or just show one if we are in debug mode and target the developers?